lib/taurus/qt/qtgui/panel/taurusmodelchooser.py

```diff
@@ -149,6 +149,8 @@ class TaurusModelChooser(TaurusWidget):
     '''
 
     updateModels = Qt.pyqtSignal('QStringList')
+    # (list<str> or QMimeData)
```

Comment:
It seems that UpdateAttrs could emit a list of str or QMimeData (as are the possible returns from getListedModels), so I'll update the signal signature, that should be enough. 
Also, this signal is only used for backwards compatibility with the old AttributeChooser ...